src/main/resources/schema/ans/0.10.3/traits/trait_gallery_properties.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.3/traits/trait_gallery_properties.json",
+  "title": "Has gallery properties",
+  "description": "An object for over-rides for images when images are used in a gallery. Example over-ride is trait_focal_point.",
```

Comment:
overrides

Reply:
Probably could use a longer description explaining how the overrides should be used